scripts/build/rollup-plugins/babel.js

```diff
@@ -0,0 +1,16 @@
+"use strict";
```

Comment:
https://github.com/rollup/rollup-plugin-babel?

Reply:
I got confused with lots of stuff on the README, just wanted to ship something.. I'll check it out again

Reply:
Also we should probably use `babel-preset-env` for this but I wanted to make follow-up PRs since this was already too big